index.html

```diff
@@ -2,13 +2,138 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta name="description" content="Bocharov design studio">
+    <meta name="keywords" content="interior design, author's works">
+    <meta name="autor" content="Anton and Irina Bocharov">
+    <meta name="vienport" content="width=device-width, initial-scale=1.0">
     <title>Моя первая страница</title>
   </head>
   <body>
     <h1>Моя первая страница</h1>
-    <hr />
+    <hr>
     <p>
       <a href="Hello/about.html">об авторе</a>
     </p>
+    <img src="./images/flower.jpg" alt="flowers">
+    <b>
```

Comment:
`<ol>` is not phrasing content. Don't use `<b>` to wrap the blocks.
Read more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b)

Reply:
I corrected.